blockchain/proxy.go

```diff
@@ -84,19 +84,14 @@ func NewProxy(args ArgsProxy) (*proxy, error) {
 		return nil, err
 	}
 
-	cacher := args.FilterQueryBlockCacher
-	if cacher == nil {
```

Comment:
because of this, all other usages of this proxy (other projects) will need to provide a cacher, even if it is not used. I liked the old variant because:
1. the older usages won't break
2. this cacher is optional, the code can work without it so it can be ommited, if desired